R/status.R

```diff
@@ -264,6 +228,51 @@ renv_status_check_consistent <- function(lockfile, library, used) {
 
 }
 
+renv_status_check_initialized <- function(project, library, lockfile) {
+
+  # only done if library and lockfile are NULL; that is, if the user
+  # is calling `renv::status()` without arguments
+  if (!is.null(library) || !is.null(lockfile))
+    return(TRUE)
+
+  # if the project is already loaded, then we can assume it's initialized
+  if (renv_project_loaded(project))
+    return(TRUE)
+
+  # resolve paths to lockfile, primary library path
+  library  <- library  %||% renv_paths_library(project = project)
+  lockfile <- lockfile %||% renv_paths_lockfile(project = project)
+
+  # check whether the lockfile + library exist
+  haslib  <- file.exists(library)
```

Comment:
Does this need `any(file.exists(library))` in case the user supplies a vector of library paths?

Reply:
Thanks! I'll make that change.